freqtrade/freqtradebot.py

```diff
@@ -175,6 +175,23 @@ def process_stopped(self) -> None:
         if self.config['cancel_open_orders_on_exit']:
             self.cancel_all_open_orders()
 
+    def check_for_open_trades(self):
+        """
+        Notify the user when the bot is stopped
+        and there are still open trades active.
+        """
+        open_trades = Trade.get_trades([Trade.is_open == 1]).all()
+
+        if len(open_trades) != 0:
+            msg = {
+                'type': RPCMessageType.WARNING_NOTIFICATION,
+                'status':  f'{len(open_trades)} open trades active.\n\n'
+                           f'Handle these trades manually on {self.exchange.name}, '
+                           f'or \'/start\' the bot again and use \'/stopbuy\' '
```

Comment:
I would recommend to use double quotes for the string - so you don't have to escape the tics:

```suggestion
                           f"or '/start' the bot again and use '/stopbuy' "
```

Obviously applies to all other lines, too...